Password Verify/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Problem
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            String pass;
+            pass = Console.ReadLine();
+
+            int a = 0;
+            int b = 0;
+            Boolean c = true;
+            Boolean num = false;
```

Comment:
Use meaningful names for these variables like,
a -> LowerCaseCount,
b -> UpperCaseCount,
c -> isValidLength
num -> containsNumber